### ISSUE TITLE
Modify the artifact manager to make newly created magnet link available to other nodes' artifact managers through a libp2p pubsub mechanism.

### ISSUE THREAD

**Issue by mgrand** [pyrsia/pyrsia#278]
This task is to be done after #276. 

Use one of the pubsub protocols provided by `libp2p` to push the magnet links created by the artifact manager to other nodes. This task includes pushing magnet links and an integration test that verifies that the magnet links are received through pubsub.

Consuming the magnet links to get artifacts through bittorrent is in the next issue #279

**Comment by efrisch** [comment]
Closing due to bittorrent stories being out of scope / replaced with bitswap.
